scripts/react-tester.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-env node */
+
+const { execSync } = require( 'child_process' );
+const { mkdirSync, rmdirSync, copyFileSync } = require( 'fs' );
+const { resolve: resolvePath } = require( 'path' );
+const satisfiesSemver = require( 'semver/functions/satisfies' );
+
+const PACKAGE_PATH = resolvePath( __dirname, '..' );
+const TESTS_PATH = resolvePath( PACKAGE_PATH, 'react-tests' );
+
+const packageInfo = require( '../package.json' );
+const availableVersions = getVersions();
+const semverRange = getReactVersion( packageInfo );
+const versionsToTest = getVersionsInRange( semverRange, availableVersions );
+
+try {
+	console.log( '--- Ultimate React Tester ---' );
+	mkdirSync( TESTS_PATH );
+	versionsToTest.forEach( testVersion );
+	rmdirSyncRecursive( TESTS_PATH );
+	console.log( 'Done without errors. Have a nice day!' );
+} catch ( error ) {
+	console.error( 'Error occured :(' );
+	console.error( error );
+	process.exit( 1 );
+}
+
+function getVersions() {
+	const commandResult = execNpmCommand( 'view react versions --json' );
+	const versions = JSON.parse( commandResult );
+
+	return versions;
+}
+
+function getReactVersion( packageInfo ) {
+	const peerDependencies = packageInfo.peerDependencies;
+	const react = peerDependencies.react;
+
+	return react;
+}
+
+function getVersionsInRange( range, versions ) {
+	return versions.filter( version => {
+		// 16.6.2 is broken.
```

Comment:
Is it some React issue or our integration? If it's our integration we should report an issue. If React maybe some external reference could be useful?

Reply:
It's React - added reference to an issue: https://github.com/facebook/react/issues/14208.